.ci/update-specs.yml

```diff
@@ -0,0 +1,65 @@
+---
+name: update specs
+
+scms:
+  githubConfig:
+    kind: github
+    spec:
+      user: '{{ requiredEnv "GIT_USER" }}'
+      email: '{{ requiredEnv "GIT_EMAIL" }}'
+      owner: elastic
+      repository: ecs-dotnet
+      token: '{{ requiredEnv "GITHUB_TOKEN" }}'
+      username: '{{ requiredEnv "GIT_USER" }}'
+      branch: main
+
+actions:
+  ecs-dotnet:
+    kind: github/pullrequest
+    scmid: githubConfig
+    sourceid: sha
+    spec:
+      automerge: false
+      labels:
+        - dependencies
+      title: 'synchronize ecs-logging spec'
```

Comment:
Is this actually working for you? 😲

Reply:
I'm using `dry-run` in my local, let me try with my fork

Reply:
https://github.com/v1v/ecs-dotnet/pull/1 nop! 

But by changing the `target` name:
- https://github.com/v1v/ecs-dotnet/pull/2

```diff
 targets:
   spec.json-update:
-    name: Update spec.json
+    name: 'synchronize ecs-logging spec'
     kind: file
     sourceid: spec.json
     scmid: githubConfig
```

Reply:
Interesting. Thanks for checking.

Reply:
Interesting. Thanks for checking.

Reply:
Interesting. Thanks for checking.